### PR TITLE
BigQuery:deprecated warnings removed

### PR DESCRIPTION
**Pull request by abhinav-qlogic**

### CODE REVIEW

File: google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java [hunk 1]
```diff
@@ -46,6 +48,8 @@
 
   private static final int MAX_INBOUND_MESSAGE_SIZE = 1024 * 1024 * 11;
 
+  private static List<Code> codeList = Arrays.asList(Code.DEADLINE_EXCEEDED, Code.UNAVAILABLE);
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java]: rename it to `retryCodeList`

**Review by rahulKQL — APPROVED** [review]
Other than that rename... everything else looks good to me

File: google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java (hunk 1 above)
Comment (abhinav-qlogic) [reply google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java]: done

File: google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java [hunk 2]
```diff
@@ -120,7 +124,7 @@ public void testCreateReadSessionSettings() {
   public void testReadRowsSettings() {
     ServerStreamingCallSettings.Builder<ReadRowsRequest, ReadRowsResponse> builder =
         EnhancedBigQueryStorageStubSettings.newBuilder().readRowsSettings();
-    assertThat(builder.getRetryableCodes()).containsAllOf(Code.DEADLINE_EXCEEDED, Code.UNAVAILABLE);
+    assertThat(builder.getRetryableCodes().containsAll(retryCodeList));
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java]: Please update this as well to `containsAtLeast` or with `isTrue()`

Comment (abhinav-qlogic) [reply google-cloud-clients/google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta1/stub/EnhancedBigQueryStorageStubSettingsTest.java]: done

File: google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java [hunk 3]
```diff
@@ -979,7 +979,7 @@ public void testDeleteTable() {
     EasyMock.expect(bigqueryRpcMock.deleteTable(PROJECT, DATASET, TABLE)).andReturn(true);
     EasyMock.replay(bigqueryRpcMock);
     bigquery = options.getService();
-    assertTrue(bigquery.delete(DATASET, TABLE));
+    assertTrue(bigquery.delete(TABLE_ID));
```
Comment (ajaaym) [inline google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java:982]: why did you change this?

**Review by ajaaym — COMMENTED** [review]
why did you combine bigquery and storage in this pr?
